compiler/one-cmds/tests/one-init-002.test

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
I have to figure out how to manually run this test. 
Any help could save my time a lot!

Reply:
(1) make a local setup with
```
NNAS_BUILD_PREFIX=build/release \
NNCC_WORKSPACE=build/release \
BUILD_TYPE=Release \
./nnas create-package --preset 20220323 --prefix build/install
```
(2) prepare test materials
```
cd build/install/bin/test
./prepare_test_materials.sh
```

(3) run test
```
bash ./one-init-002.test
```

Reply:
In my case for (3), I commented out the tests except what I want in `build/install/test/runtestall.sh` and ran the script.

Reply:
@seanshpark,
For (2),
I couldn't find `build/install/bin/test/prepare_test_materials.sh`.
Should I run `build/install/test/prepare_test_materials.sh`?

```
$ find build  -name prepare_test_materials.sh
build/install/test/onnx-operations/prepare_test_materials.sh
build/install/test/prepare_test_materials.sh
```